cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/PartitionedFileSetArguments.java

```diff
@@ -170,4 +172,15 @@ public static Comparable convertFieldValue(String where, String kind, String fie
                       where, kind, stringValue, fieldName, fieldType.name()), e);
     }
   }
+
```

Comment:
This is in API, so why is there no Javadoc?
Especially, you should explain what happens if both an input partition filter, and one or more input partitions are specified.